app/mailers/user_mailer.rb

```diff
@@ -1,7 +1,7 @@
 class UserMailer < ActionMailer::Base
   include MailerHelper
 
-  default from: lambda { email_from }
+  default from: email_from()
```

Comment:
Should this be a `Proc`, as in [noisy_batch_invitation.rb](https://github.com/alphagov/signonotron2/pull/352/files#diff-e0c9d29652611247039064bbd37e2c6dR4)?